docs/guide.rst

```diff
@@ -1099,6 +1096,30 @@ For example, to treat warnings as errors and run in nit-picky mode:
 This Nox session always runs with the current major release of Python.
 
 
+.. _The docs-build session:
+
+The docs-build session
+----------------------
+
+Build the documentation using the Nox session ``docs-build``:
+
+.. code:: console
+
+   $ nox --session=docs-build
+
+The docs session runs the command ``sphinx-build``
+to generate the HTML documentation from the Sphinx directory.
+
+Use the ``--`` separator to pass additional options to either tool.
+For example, to treat warnings as errors and run in nit-picky mode:
+
+.. code:: console
+
+   $ nox --session=docs-build -- -W -n docs docs/_build
```

Comment:
We can omit the example here, since it is already included in the section about the `docs` session.

```suggestion
```